### DISCUSSION TITLE
TuringCourses/major_mandatory/computer_architecture/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#85] [General]
# TuringCourses/major_mandatory/computer_architecture/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major_mandatory/computer_architecture/

**Comment by Unc1eW4ng** [comment]
准备的时候最好能找同学要到不同老师的复习ppt，统筹起来准备比较好。22级的选择题考了相当多的cache相关的内容，会细致到具体的cache技术比如multibank cache，pipeline cache等对于cache的哪些属性是正影响还是负影响。cache一致性也是相当困难的一部分，但是这次考察的比较简单。
